pkg/deployer/clusterapiservertemplate.go

```diff
@@ -153,13 +153,13 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: apiserver
-  namespace: default
+  namespace: {{ .Namespace }}
 ---
 apiVersion: apps/v1beta1
```

Comment:
`etcd` service after this manifest should also be deployed in `{{ .Namespace }}` instead of default.

Reply:
Thanks for catching that!